### ISSUE TITLE
make wssh available through pypi

### ISSUE THREAD

**Issue by amalrosh12** [progrium/wssh#27]
this will make wssh even easier to install (pip install wssh)

**Comment by linuxlewis** [comment]
👍

**Comment by arpit1997** [comment]
Yeah @amalrosh12 . I should be made available through [PyPi](https://pypi.python.org/)
as this application is somewhat usable and has good stars. PyPi is the most usable package manager so it should be deployed on the same. There is a good tutorial by [travis-ci](http://travis-ci.org) [here](https://docs.travis-ci.com/user/deployment/pypi/) :smile_cat: :smile_cat:
